Makefile

```diff
@@ -138,30 +167,33 @@ run-ci:
 
 run-headless: ## Run the Docker container for headless machines
 run-headless: CONTAINER_NAME ?= garage-headless
+run-headless: user ?= $$USER
 run-headless: build-headless
 	docker run \
 		-it \
 		--rm \
-		-v $(DATA_PATH)/$(CONTAINER_NAME):/root/code/garage/data \
+		-v $(DATA_PATH)/$(CONTAINER_NAME):/home/$(user)/code/garage/data \
```

Comment:
you are not guaranteed that this is where garage code (or even home directories) are.

i think you should just put it in $(PWD)/garage_data or something.

Reply:
Not sure I understand this. Do you mean change it to 
`-v $(PWD)/garage_data:/home/$(user)/code/garage/date`